lib/game_creator.rb

```diff
@@ -0,0 +1,50 @@
+require_relative './game'
+require_relative './game_setting/game_setting_setter'
+require_relative './game_setting/setting_types'
+require_relative './persister/persister'
+require_relative './repository/local'
+
+class GameCreator
+    include SettingTypes
+
+    def initialize(repository_type:, game_io: GameIO.new(presenter: ConsoleIO.new))
+        @persister = Persister.new(repository_type: repository_type)
+        @game_settings = GameSettingSetter.new(game_io: game_io)
+        @suspended_games = persister.get_list_of_suspended_games
+    end
+
+    def create
+        game_setting_input = get_new_or_suspended_game_input if persister.has_saved_games?(suspended_games)
+
+        return resume_game(game_setting_input) if is_a_suspended_game_id?(game_setting_input)
+        return new_game 
+    end
+
+    private 
+    attr_accessor :persister, :game_settings, :game, :suspended_games
+
+    def get_new_or_suspended_game_input
+        suspended_game_ids = persister.suspended_game_ids(suspended_games)
+        new_or_suspended_game_settings = GameSettings.new_or_suspended_game(suspended_games, suspended_game_ids)
+        setting_output = game_settings.create_settings(settings: new_or_suspended_game_settings)
+        setting_output.fetch(SettingTypes::NEW_OR_SUSPENDED_GAME)
+    end
+
+    def is_a_suspended_game_id?(input)
+        input.is_a?(Integer)
```

Comment:
Seems strange to me that any integer input would satisfy this since it sounds like any number would be a valid suspended game id.

Reply:
so there's already a validation check when grabbing user input -- this check is whether the input is a integer, in which case the user is asking for a suspended game, OR, "N" (string) for a New Game

Reply:
the naming is confusing cause it seems its a validation.
maybe pull this into validation file, so its grouped with the others.